### DISCUSSION TITLE
Building Hybrid Search Engine using Open AI LLMs ,  Langchain and Milvus.

### DISCUSSION THREAD

**Discussion by Pranav-NG** [milvus-io/milvus#27002] [Q&A and General discussion]
Hi Team , 

I just wanted to know how to build an Hybrid Search Engine using LLM.

Langchain has options for self query retriever for databases other than milvus. 
https://python.langchain.com/docs/modules/data_connection/retrievers/self_query/

How to use the incorporate the same with Milvus. ?

Please guide , 

Thanks, 
Pranav,

**Comment by yhmo** [comment]
This article is helpful: https://milvus.io/docs/integrate_with_langchain.md
